src/main/java/de/clearit/kindergarten/appliance/vendor/VendorEditorModel.java

```diff
@@ -114,7 +115,9 @@ public void addVendorNumber(ActionEvent e) {
     getBean().getVendorNumbers().add(vendorNumberBean);
     // Reset input field
     getVendorNumberFieldModel().setValue(null);
-
+    // Set Focus on TextField

```

Comment:
Den Fokuswechsel würde ich eher in der View machen, das Model hat ja damit nix zutun